projects/ngrx.io/content/guide/signals/signal-store/entity-management.md

```diff
@@ -384,3 +384,41 @@ export const TodosStore = signalStore(
 );
 
 </code-example>
+
+## Private Entity Collections
+
+Private entity collections are defined by using the `_` prefix for the collection name.
+
+```ts
+const todoConfig = entityConfig({
+  entity: type<Todo>(),
+  // 👇 private collection
+  collection: '_todo',
```

Comment:
Should we add a JSDoc comments for the `collection` property to mention this behavior?
I'm thinking that new devs wouldn't expect this behavior by naming a collection.

Reply:
Definitely! I'm going to add JSDoc for all public APIs.